app/full_text_search.py

```diff
@@ -171,10 +199,18 @@ def create_document(record_id, repo, **kwargs):
     fields.append(appengine_search.TextField(name='record_id', value=record_id))
 
     # Add name and location romanized by unidecode
+    name_fields = ['given_name', 'full_name', 'family_name', 'alternate_names']
     for field in kwargs:
         romanized_value = script_variant.romanize_word(kwargs[field])
         fields.append(
             appengine_search.TextField(name=field, value=romanized_value))
```

Comment:
location: used, name: not necessary.
- >changed: add only location(without_for_rank)